services/grafana/auth_server_test.go

```diff
@@ -208,8 +208,8 @@ func TestAuthServerAuthenticate(t *testing.T) {
 		"/v1/Updates/Status":                               none,
 		"/v1/Settings/Get":                                 admin,
 		"/v1/AWSInstanceCheck":                             none,
-		"/v1/Platform/SignUp":                              admin,
-		"/v1/Platform/Connect":                             admin,
+		"/v1/Platform/SignUp":                              viewer,
+		"/v1/Platform/Connect":                             viewer,
```

Comment:
wait, I think only Grafana Admin should be able to connect PMM to Portal.

Reply:
changed this, now connect and disconnect have admin and tickets endpoint viewer